organisaatio-ui/src/components/Sivut/Ryhmat/Ryhmat.tsx

```diff
@@ -52,51 +50,46 @@ const Ryhmat: React.FC = (props) => {
                     </a>
                 );
             },
-            collapse: true,
+            accessor: (values: any) => {
+                return values.nimi[language] || values.nimi.fi || values.nimi.sv || values.nimi.en || '';
```

Comment:
This looks bit ugly. ok it might work now but what about if set of supported languages change for some reason? It _might_ make sense to extract this kind of logic to some common place

Reply:
Yes indeed. This nimikikkailu is a little bit of a mystery to me so there is plenty of things to be improved im sure.